projects/showcase/.eslintrc.json

```diff
@@ -0,0 +1,52 @@
+{
+  "extends": "../../.eslintrc.json",
```

Comment:
It extends from this file, but the content is quite similar. Can we clean the shared parts?

Reply:
Removed duplicities